pages/index.html

```diff
@@ -0,0 +1,248 @@
+                                    <p>You will have the chance to meet and greet with all of the Athletes, network and learn.</p>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="my-4" id="join-btn">
+            <button type="button" class="d-md-none">Join EXPO SANTA FE 2021</button>
+            <p class="d-none d-md-flex lato-font">SEE THE WHOLE PROGRAM</p>
+        </div>
+    </section>
+    <section id="featured-athletes">
+        <div class="container">
+            <div class="d-flex align-items-center flex-column">
+                <h2 class="mb-0 mt-4">Featured Athletes</h2>
+                <hr class="half-border">
+            </div>
+            <div class="row row-cols-1 row-cols-md-2 gx-3 gy-3 gx-md-4 gy-md-4" id="list-athlete">
+                <div class="col featured-athlete d-none">
+                    <div class="d-flex align-items-center">
+                        <img class="athlete-img me-3" src="exampleSRC" alt="Athlete Photo">
+                        <div class="info-athlete">
+                            <h3 class="oswald-font">Example Name</h3>
+                            <h4 class="lato-font">Example for red text</h4>
+                            <hr class="mt-0 mb-2">
+                            <p class="lato-font">Lorem Ipsum is that it has a more-or-less normal distribution of letters</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="partners-section py-4 d-none d-md-flex flex-column">
+        <div class="d-flex flex-column align-items-center">
+            <h2 class="oswald-font mb-0 text-white">Partner</h2>
+            <hr class="half-border"> 
+        </div>
+        <div class="container">
+            <div class="row row-cols-1 row-cols-md-2 gx-md-0 g-4">
+                <div class="col d-flex justify-content-around align-items-center">
+                    <i class="fab fa-spotify"></i>
+                    <i class="fab fa-amazon"></i>
+                    <i class="fab fa-google-plus-square"></i>
+                    <span class="oswald-font"> NAVER</span>
+                </div>
+                <div class="col d-flex justify-content-around align-items-center">
+                <i class="fab fa-airbnb"></i>
+                <span class="oswald-font">daumkako</span>
+                <i class="fab fa-affiliatetheme"></i>
+                </div>
+            </div>
+        </div>
+    </section>
+    <footer class="d-flex justify-content-center py-2" id="footer-home">
+        <p class="mt-3 oswald-font text-center" id="footer-info"> 
+            <span>&#169;</span> Santa Fe Expo 2021 <br>
+            License and Credits to Cindy Shin: <a href="https://www.behance.net/adagio07">Behance</a> 
+        </p>
+    </footer>
```

Comment:
- [ ] On the mobile view of the footer on the home page the background color is white. Please kindly fix that.

Reply:
Hi @emmanuellekamwa, I don't understand fully this point the background actually is gray, or do you mean that it should be white? , please can you make this point more clear. Thank you